S2VX.Game/Editor/EditorScreen.cs

```diff
@@ -98,100 +99,116 @@ public class EditorScreen : Screen {
-            {
-                Track,
-                Story,
-                NoteSelectionIndicators,
-                ToolContainer,
-                NotesTimeline,
-                new EditorInfoBar(),
-                new BasicMenu(Direction.Horizontal, true)
+        private BasicMenu CreateMenu() =>
+            new BasicMenu(Direction.Horizontal, true) {
+                BackgroundColour = Color4.Black.Opacity(0.9f),
+                Width = 1,
+                RelativeSizeAxes = Axes.X,
+                Items = new[]
                 {
-                    BackgroundColour = Color4.Black.Opacity(0.9f),
-                    Width = 1,
-                    RelativeSizeAxes = Axes.X,
-                    Items = new[]
+                    new MenuItem("Project")
                     {
-                        new MenuItem("Project")
+                        Items = new[]
                         {
-                            Items = new[]
-                            {
-                                new MenuItem("Refresh (Ctrl+R)", ProjectRefresh),
-                                new MenuItem("Save (Ctrl+S)", ProjectSave),
-                                new MenuItem("Play (G)", ProjectPlay),
-                                new MenuItem("Quit (Esc)", ProjectQuit),
-                            }
-                        },
-                        new MenuItem("Edit")
+                            new MenuItem("Refresh (Ctrl+R)", ProjectRefresh),
+                            new MenuItem("Save (Ctrl+S)", ProjectSave),
+                            new MenuItem("Play (G)", ProjectPlay),
+                            new MenuItem("Quit (Esc)", ProjectQuit),
+                        }
+                    },
+                    new MenuItem("Edit")
+                    {
+                        Items = new[]
                         {
-                            Items = new[]
-                            {
-                                new MenuItem("Undo (Ctrl+Z)", EditUndo),
-                                new MenuItem("Redo (Ctrl+Shift+Z)", EditRedo)
-                            }
-                        },
-                        new MenuItem("View")
+                            new MenuItem("Undo (Ctrl+Z)", EditUndo),
+                            new MenuItem("Redo (Ctrl+Shift+Z)", EditRedo)
+                        }
+                    },
+                    new MenuItem("View")
+                    {
+                        Items = new[]
                         {
-                            Items = new[]
-                            {
-                                new MenuItem("Command Panel (Ctrl+1)", ViewCommandPanel),
-                            }
-                        },
-                        new MenuItem("Playback")
+                            new MenuItem("Editor Interface (H)", ViewEditorUI),
+                            new MenuItem("Command Panel (Ctrl+1)", ViewCommandPanel),
```

Comment:
I kinda want to change this from Ctrl+1 to some single letter hotkey since it feels like an integral/essential part of story making that is used often.

Reply:
I will change it to just C